pkgs/applications/networking/browsers/vieb/default.nix

```
@@ -34,6 +34,11 @@ mkYarnPackage rec {
   };
 
   postInstall = ''
+    unlink $out/libexec/vieb/deps/vieb/node_modules
+    ln -s $out/libexec/vieb/node_modules $out/libexec/vieb/deps/vieb/node_modules
+
+    find $out/libexec/vieb/node_modules/7zip-bin -name 7za -exec ln -s -f ${p7zip}/bin/7z {} ';'
```

It looks like 7za and 7z are not the same?
```suggestion
    find $out/libexec/vieb/node_modules/7zip-bin -name 7za -exec ln -s -f ${p7zip}/bin/7za {} ';'
```

---

Oops, sorry about that. Just fixed it.